piv/src/main/java/com/yubico/yubikit/piv/jca/PivKeyPairGeneratorSpi.java

```diff
@@ -71,13 +71,13 @@ public KeyPair generateKeyPair() {
             BlockingQueue<Result<KeyPair, Exception>> queue = new ArrayBlockingQueue<>(1);
             provider.invoke(result -> queue.add(Result.of(() -> {
                 PivSession session = result.getValue();
-                PublicKey publicKey = session.generateKey(spec.slot, spec.keyType, spec.pinPolicy, spec.touchPolicy);
+                PublicKey publicKey = session.generateKey(spec.slot, spec.keyType, spec.pinPolicy, spec.touchPolicy); //TODO Dont use deprecated function
```

Comment:
Why leave this as a TODO instead of just calling generateKeyValues().toPublicKey()?